### PR TITLE
Update rate limits docs

### PR DESCRIPTION
**Pull request by xavierdecoster**
Based on feedback: Rate limiter error is incorrect #934

### CODE REVIEW

**Comment by agr** [comment]
Can we set up rate limiter to produce 429?

**Comment by xavierdecoster** [comment]
@agr the reason is that they are different types of configuration rules in APIM. 
One is rate limiting requests per minute, the other is a quota on number of requests per hour. 

The first does not support a `renewal-period` of an hour (undocumented but confirmed by APIM folks) and doesn't meet the rate-limiting requirements for those particular API calls: it's a [`rate-limit-by-key`](https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#LimitCallRateByKey), returning 429 when excceeded. 

The latter supports a `renewal-period` of an hour, and is a [`quota-by-key`](https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#SetUsageQuotaByKey), returning 403 when exceeded. 

See also https://docs.microsoft.com/en-us/azure/api-management/api-management-sample-flexible-throttling

File: docs/api/rate-limits.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ The NuGet.org API enforces rate limiting to prevent abuse. Requests that exceed
     }
   ~~~
 
+In addition to request throttling using rate limits, some API's also enforce quota. Requests that exceed the quota return the following error:
```
Comment (loic-sharma) [inline docs/api/rate-limits.md]: `API's` -> `APIs`

Comment (xavierdecoster) [reply docs/api/rate-limits.md]: Thx! Fixed.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/xavierdecoster-patch-1/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-941) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201806250818125160-941%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Aycfc0b2fVgUFqEA3zht%2B4nanz4uHrmYbC5JdNiPqGI%3D&st=2018-06-25T08%3A14%3A10Z&se=2018-07-26T08%3A19%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by xavierdecoster** [comment]
Any reviewers want to sign off on this one? Or is there any additional feedback?

**Comment by karann-msft** [comment]
Adding @CamSoper

**Pull request merged**